app/controllers/home_controller.rb

```diff
@@ -19,15 +19,17 @@ def dashboard
       user_id: current_user.id,
       recent: true,
       include_private: true,
-      include_adult: !current_user.sfw_filter?
+      include_adult: !current_user.sfw_filter?,
```

Comment:
Is there a rule demanding that there be a trailing comma?  I should probably disable that